testing/tests/DevExpress.ui.events/events.utils.tests.js

```diff
@@ -583,6 +583,16 @@ QUnit.module('skip mousewheel event test', () => {
             $element.remove();
         }
     });
+
+    test('needSkipEvent returns false for nested in .dx-skip-gesture-event dropdownlist popup wrapper dxmousewheel (T1000311)', function(assert) {
+        const $element = $('<div class=\'dx-skip-gesture-event\'></div>');
```

Comment:
Did u forget to append if ti `$('#qunit-fixture')`? Looks like evergreen

Reply:
no, it's good